### PR TITLE
Check if product node exists for sku in  cart.

### PR DESCRIPTION
**Pull request by joshirohit100**

### CODE REVIEW

**Comment by sdelbosc** [comment]
I wonder if we should log the cases

File: modules/sku/src/AcquiaCommerce/SKUPluginBase.php [hunk 1]
```diff
@@ -163,9 +163,14 @@ public function cartName(SKU $sku, array $cart, $asString = FALSE) {
...
-      $url = $display_node->toUrl();
-      $link = Link::fromTextAndUrl($cartName, $url);
-      $cartName = $link->toRenderable();
+      if ($display_node instanceof Node) {
+        $url = $display_node->toUrl();
+        $link = Link::fromTextAndUrl($cartName, $url);
+        $cartName = $link->toRenderable();
+      }
+      else {
+        \Drupal::logger('acq_sku')->info('Parent product for the sku: @sku seems to be unavailable.', ['@sku' => $sku->getSku()]);
```
Comment (miromichalicka) [inline modules/sku/src/AcquiaCommerce/SKUPluginBase.php]: logger should be acm_sku

Comment (joshirohit100) [reply modules/sku/src/AcquiaCommerce/SKUPluginBase.php]: PR Updated! Thanks!

**Comment by miromichalicka** [comment]
@joshirohit100 please update also the rest of loggers :)

**Comment by joshirohit100** [comment]
@miromichalicka I checked and looks like loggers are already updated with the correct name. I see logger with **acm**, **acm_sku**, **acm_checkout**, **acm_promotion**, **acm_exception** etc. Can you pls provide more info on this.

**Comment by miromichalicka** [comment]
Ok, my bad.

**Pull request merged**